src/Retrofit/PaymentIntentNextAction.php

```diff
@@ -0,0 +1,12 @@
+<?php
+declare(strict_types = 1);
+
+namespace Spaze\PHPStan\Stripe\Retrofit;
+
+/**
+ * @property StripeObject $redirect_to_url
+ * @property 'redirect_to_url'|'use_stripe_sdk'|'alipay_handle_redirect'|'oxxo_display_details'|'verify_with_microdeposits' $type
```

Comment:
TIL that literals can be [written like this](https://phpstan.org/writing-php-code/phpdoc-types#literals-and-constants), nice!

Reply:
if they were defined in constants you could use `Foo::TYPE_*` which is more interesting.